_1327/templates/base.html

```diff
@@ -95,6 +95,18 @@
 							</ul>
 						</li>
 					{% endif %}
+					{# Search form #}
```

Comment:
please add some javascript that automatically focuses the search text field on opening the popover